### PR TITLE
Add a hidden option for -H

### PR DESCRIPTION
**Pull request by blampe**
@prashantv

### CODE REVIEW

File: options.go [hunk 1]
```diff
@@ -64,6 +64,9 @@ type TransportOptions struct {
 
 	// benchmarking is a private flag set when a transport is required for benchmarking.
 	benchmarking bool
+
+	// Alias for tcurl compatibility.
+	H stringAlias `short:"H" hidden:"true"`
```
Comment (prashantv) [inline options.go]: should we also add `hostfile` or whatever the long version was here?

File: options.go [hunk 2]
```diff
@@ -64,6 +64,9 @@ type TransportOptions struct {
 
 	// benchmarking is a private flag set when a transport is required for benchmarking.
 	benchmarking bool
+
+	// Alias for tcurl compatibility.
```
Comment (abhinav) [inline options.go]: alias of what though?

**Comment by prashantv** [comment]
lgtm

File: options.go (hunk 1 above)
Comment (blampe) [inline options.go]: sure!

**Comment by abhinav** [comment]
LGTM

**Pull request merged**